replication/binlogsyncer.go

```diff
@@ -783,13 +803,6 @@ func (b *BinlogSyncer) parseEvent(s *BinlogStreamer, data []byte) error {
 	return nil
 }
 
-func (b *BinlogSyncer) getGtidSet() GTIDSet {
```

Comment:
moved `getGtidSet` to local function to emphasize that it is being used only inside single function